week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// This program will join together two or more strings.
 // - How many parameters does the function take?
+// The funtion takes 2 parameters.
 // - What is the function name?
+// the function name is s.
 // - Where is the function called? with what parameters?
+// The function is called at line 5.
 // - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// the function return "codeyour".
+// - How can this function be improved?
+// this function can be improved strings with space inside the parenthese of concat.
```

Comment:
It could be better to add a space, but it's hard to say without knowing what the function will be used for. There's one other thing I would definitely want to improve. Remember that it is not only important that the code works well but also that it is easy for people reading it to understand. Do you know what I am talking about?